protocol/contact.go

```diff
@@ -39,6 +44,20 @@ func (c *Contact) CanonicalName() string {
 }
 
 func (c *Contact) CanonicalImage() string {
+	if largeImage, ok := c.Images[largeImageType]; ok {
+		imageBase64, err := largeImage.GetDataURI()
+		if err == nil {
+			return imageBase64
+		}
+	}
+
+	if thumbImage, ok := c.Images[thumbnailImageType]; ok {
+		imageBase64, err := thumbImage.GetDataURI()
+		if err == nil {
+			return imageBase64
+		}
+	}
+
```

Comment:
FYI this const already exists.

`"github.com/status-im/status-go/images"`

```suggestion
	if largeImage, ok := c.Images[images.LargeDimName]; ok {
		imageBase64, err := largeImage.GetDataURI()
		if err == nil {
			return imageBase64
		}
	}

	if thumbImage, ok := c.Images[images.SmallDimName]; ok {
		imageBase64, err := thumbImage.GetDataURI()
		if err == nil {
			return imageBase64
		}
	}

```